fee_allocator/accounting/core_pools.py

```diff
@@ -82,18 +84,42 @@ def __init__(self, data: PoolFeeData, chain: CorePoolChain):
         self.__dict__.update(vars(data))
         self.chain = chain
 
+        self.is_alliance_pool = self._check_if_alliance_pool()
+        self.alliance_fee_config = self._get_alliance_fee_config() if self.is_alliance_pool else None
+        self.is_alliance_non_core_pool = self._is_alliance_non_core_pool()
+
         self.original_earned_fee_share = Decimal(0)
         self.earned_fee_share_of_chain_usd = self._earned_fee_share_of_chain_usd()
         self.total_to_incentives_usd = self._total_to_incentives_usd()
         self.to_aura_incentives_usd = self._to_aura_incentives_usd()
         self.to_bal_incentives_usd = self._to_bal_incentives_usd()
         self.to_dao_usd = self._to_dao_usd()
         self.to_vebal_usd = self._to_vebal_usd()
+        self.to_partner_usd = self._to_partner_usd() if self.is_alliance_pool else Decimal(0)
```

Comment:
`if` `else` looks to be redundant here

Reply:
good catch, ya that logic is already contained in the `_to_partner_usd` method: [2624dd6](https://github.com/BalancerMaxis/protocol_fee_allocator_v2/pull/110/commits/2624dd6422a89c6b22f9c303b5cc8e1ba81ef1e2)